api/server/router/swarm/cluster_routes.go

```diff
@@ -372,6 +372,10 @@ func (sr *swarmRouter) createSecret(ctx context.Context, w http.ResponseWriter,
 	if err := json.NewDecoder(r.Body).Decode(&secret); err != nil {
 		return err
 	}
+	version := httputils.VersionFromContext(ctx)
+	if secret.Templating != nil && versions.LessThan(version, "1.36") {
```

Comment:
oh, dang! this needs to be 1.37 now (https://github.com/moby/moby/pull/33922 was merged)

Reply:
I'm ok with fixing that in a follow-up